Sources/SpeziOnboarding/ConsentView/ConsentDocument.swift

```diff
@@ -47,14 +50,22 @@ public struct ConsentDocument: View {
     private var nameView: some View {
         VStack {
             Divider()
-            
-            NameFields(
-                name: $name,
-                givenNameField: givenNameField,
-                familyNameField: familyNameField
-            )
+            Grid(horizontalSpacing: 15) { // optional horizontal spacing
```

Comment:
Some leftovers from the example.

```suggestion
            Grid(horizontalSpacing: 15) {
```

Reply:
Thank you, good catch!